README.md

```diff
@@ -35,59 +36,64 @@ When user back to you page with JWT token, validate it and login user.
 npm i --save silkey-sdk
 ```
 
-### Integration
+### Silkey Integration Guide
 
-#### on signin page
+#### Needed Preperations:
 
-```    
-    const callerParams = {
-      // make sure you support both GET and POST redirection,
-      redirectUrl: 'https://your-registered-domain/redirect-path', 
-      cancelUrl: 'https://your-registered-domain/cancel',
-      refId: 'any',
-      scope: 'email'
-    }
-    
-    const requestParams = await silkeySdk.generateSSORequestParams(callerPrivateKey, callerParams)
+1.  Create a ethereum wallet for the program, save the private key
```

Comment:
we need to explain that this is for domain, not for a program and how this key will be used

Reply:
I'll edit the wording to specify that the wallet is linked to the domain when registered with apollo.